crates/saya/core/src/lib.rs

```diff
@@ -196,30 +198,39 @@ impl Saya {
             state_updates: state_updates_to_prove,
         };
 
-        println!("Program input: {}", new_program_input.serialize()?);
-
-        // let to_prove = ProvedStateDiff {
-        //     genesis_state_hash,
-        //     prev_state_hash: prev_block.header.header.state_root,
-        //     state_updates: state_updates_to_prove,
-        // };
+        let world_da = new_program_input.da_as_calldata(self.config.world_address);
+        let world_da_printable: Vec<String> = world_da.iter().map(|x| x.to_string()).collect();
+        trace!(target: "saya_core", "World DA {world_da_printable:?}.");
```

Comment:
If you can put the `saya_core` into a constant `LOG_TARGET` as we have across the stack it could be great. But can definitely be done later.

Reply:
fixed